expect.kt/src/test/kotlin/com/nhaarman/expect/SetMatcherTest.kt

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2019 Uziel Sulkies
```

Comment:
I'm not sure if this is the convention for code contributors.
Please tell me if change is needed.